src/components/admin/ScraperDashboard.tsx

```
@@ -0,0 +1,230 @@
+               type="number" 
+               value={pageLimit} 
+               onChange={(e) => setPageLimit(Number(e.target.value))}
+               className="w-full p-2 border rounded dark:bg-gray-700"
+             />
+             <div className="text-xs text-gray-500 mt-1">
+               Default: NEW=3, BACKFILL=10
+             </div>
+          </div>
+          
+          <div>
+             <label className="block text-sm font-medium mb-1">Interval (ms)</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)